client/state/post-types/schema.js

```diff
@@ -0,0 +1,23 @@
+export const items = {
+	type: 'object',
+	additionalProperties: false,
+	patternProperties: {
+		'^\\d+$': {
+			type: 'object',
+			additionalProperties: {
+				type: 'object',
+				required: [ 'name' ],
+				properties: {
+					name: { type: 'string' },
+					label: { type: 'string' },
+					labels: { type: 'object' },
+					description: { type: 'string' },
+					map_meta_cap: { type: 'boolean' },
```

Comment:
>Just noting that this schema changes a little bit in the corresponding WP-API endpoint.

A good reason for schemas! Since this should then be captured by a failing test if `wpcom`'s `postTypesList` changes under our feet.